modules/apps/product-navigation/product-navigation-product-menu-web/src/main/resources/META-INF/resources/portlet/pages_tree.jsp

```diff
@@ -189,10 +189,12 @@ LayoutsTreeDisplayContext layoutsTreeDisplayContext = new LayoutsTreeDisplayCont
 	module="js/PagesTreeEventHandler.es"
 />
 
-<aui:script require="metal-dom/src/all/dom as dom">
+<aui:script require="frontend-js-web/liferay/delegate/delegate.es as delegateModule">
 	var layoutsTree = document.getElementById('<portlet:namespace />layoutsTree');
 
-	var viewCollectionItemsActionOptionQueryClickHandler = dom.delegate(
+	var delegate = delegateModule.default;
```

Comment:
Why the intermediate `var` rather than simply using here `delegateModule.default(...)`?

I think I saw similar kind of feedback in a similar PR?

We're at most using `delegate` once in all these JSPs, so the additional var seems rather pointless.

Ideally, we wouldn't even have it here, but it is what it is for now. If we wanted to make this simpler, we'd expose delegate globally, which has a different set of tradeoffs.

Reply:
If I remember correctly, you, @jbalsas commented early in this process that `delegate.default()` isn't nice, ever since then I started renaming it, it's in tons of files atm and no one complained so far.

Reply:
Pics or didn't happen!

Reply:
You might be totally right anyways

Reply:
![](https://media0.giphy.com/media/BwP6oBTVT5oC4/giphy.gif)